### PR TITLE
Name the Main and GPU thread to make debugging easier.

### PR DESCRIPTION
**Pull request by FernandoS27**

### CODE REVIEW

**Comment by jroweboy** [comment]
i tried, but failed to resist the bikeshed. i don't like putting yuzu in the name cause you already know its a yuzu thread. but even if you insist to keep it, its `yuzu` not `Yuzu` (sorry!)

File: src/core/core.cpp [hunk 1]
```diff
@@ -88,6 +89,8 @@ struct System::Impl {
     ResultStatus RunLoop(bool tight_loop) {
         status = ResultStatus::Success;
 
+        Common::SetCurrentThreadName("Yuzu Main Thread");
```
Comment (bunnei) [inline src/core/core.cpp]: Can just be "main" IMO

File: src/video_core/gpu_thread.cpp [hunk 2]
```diff
@@ -19,6 +20,7 @@ static void RunThread(VideoCore::RendererBase& renderer, Tegra::DmaPusher& dma_p
 
     // Wait for first GPU command before acquiring the window context
     state.WaitForCommands();
+    Common::SetCurrentThreadName("Yuzu GPU Thread");
```
Comment (bunnei) [inline src/video_core/gpu_thread.cpp]: Can just be "gpu" IMO

File: src/core/core.cpp (hunk 1 above)
Comment (FernandoS27) [reply src/core/core.cpp]: Would confuse with QT main (it's called main thread)

**Comment by FernandoS27** [comment]
> i tried, but failed to resist the bikeshed. i don't like putting yuzu in the name cause you already know its a yuzu thread. but even if you insist to keep it, its `yuzu` not `Yuzu` (sorry!)

I insist as it may confuse with the other threads.

**Comment by bunnei** [comment]
@FernandoS27 needs a rebase

**Comment by FernandoS27** [comment]
Comments addressed.

**Comment by bunnei** [comment]
@FernandoS27 MinGW errors are real

**Comment by bunnei** [comment]
@FernandoS27 reminder to address above comment

**Comment by FernandoS27** [comment]
@bunnei I'll look into this in the future. Closing for the time being.

**Pull request closed without merging**